### DISCUSSION TITLE
player v1.2 suddenly are malfunction in chrome normal mode

### DISCUSSION THREAD

**Discussion by efairylapis** [rutilicus/youtube-custom-music-player#6] [Q&A] [answered]
Good day rutilicus and guys

this is my first post in github, if had any not claer on my describe please feel free to let me know.

recenlty I found a player deploy by me was malfunction and this player are deploy at 2023 may alredy update to version 1.2 since it release.

there had some stiuation happened on the player before 1 or 2 weeks ago this player are working awesome.

below is what stiuation I can see.

player v1.2 are working well in 

- chrome incognito mode
- firefox normal mode.
- Microfost edge normal mode with all the same extension from chrome.

malfunction

- chrome normal mode
- disable all the extension in chrome normal mode、clear cookie, logging with YouTube account as YouTube Premium subscribe.

testing enviorment are below

OS: windows 10 pro 22H2

browser

chrome: version 120.0.6099.225 64bit
- under incognito mode: player are working awesome.
- under normal mode: malfunction, even disable all the extension、clear cookie、loging youtube account with YouTube Premium subscribe.

Microfost edge: version 120.0.2210.144 64bit
- edge normal mode:player are working awesome. install and enable all the same extension from chrome browser.

Firefox:verison 121.0.1 64bit
- normal mode: player are working awesome with some extension

when player had operational issues under chrome browser normal mode will had some stiuation below.

1.can not play the song from singing stream since correctlly timestamp, and timer also disappear from the previous、next and play button.

2.some time player will skip so many song until one of song in the playlist.

3.player functions properly for standalone cover song video(stimestap begin 0), but encounters issues when playing songs from singing stream with specify, those stiuation are happended in same player.

player in below video is deploy by me at 2023 may ( working well one or two weeks ago ) and compare with normal and incognito mode.

https://www.youtube.com/watch?v=ZwbnNtMcfjk

right side are chrome browser under Incognito mode and player are running awesome
- chrome: version 120.0.6099.225 64bit

left side are chrome browser under normal mode 
- disable all the extension
- clear cookie
- loging youtube account with YouTube Premium subscribe

and I also deploy original youtube-custom-music-player version 1.2 with default csv file again,
had same problem, I was guessing maybe there had some policy change in chrome browser that make youtube-custom-music-player malfunction?

**Comment by rutilicus** [comment]
I installed Chrome newly in my development environment, but the problem did not occur...
Chrome on my android phone occurred the problem, so something might go wrong in previous used environment.

I made issue #8, but don't hold your breath.

**Reply by efairylapis** [reply]
thank you for your qucik reply

hope my first post here and not mother language English didn't had any impolite to you.

as you say, yes the problem also occur on my android phone's chrome browser
and I also test in windows 7 64bit with last update chrome browser,
also occur same problem.

**Reply by rutilicus** [reply]
I'm debugging with Chrome on Android.
The result of loadVideoById function in YouTube IFrame Player API seems to be changed in not working environment.
I found a tweet that the startSeconds argument seems to be ignored in live streaming archive.

Edge 120.0.2210.144 and Firefox 121.0.1 occurred the same problem in my environment.

**Reply by efairylapis** [reply]
thank you for your reply to me.
interesting in my here
player under Edge 120.0.2210.144 and Firefox 121.0.1 are functional.
if rutilicus need player deploy by me for debugging please feel free to let me know.

in this question from me, also curious about if I post my 推し’ｓ　player it's will be bother Vtuberさん？
but in another hand, for techanical discussion I must post link...

**Reply by rutilicus** [reply, marked as answer]
I released V1.21 for fixing the issue by managing player loading state.
Please replace songapp_main.js. And if there is some other issue, please report.

https://github.com/rutilicus/youtube-custom-music-player/releases/tag/V121

**Reply by efairylapis** [reply]
thank you for your help
if I found any issue will report to you here

**Reply by efairylapis** [reply]
I alreday update songapp_main.js to v1.21
until now is working very well
thank you for your help and appreciate your work on this.
本当にありがとうございます
